src/test/java/core/basesyntax/service/fileservice/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.fileservice;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static FileReaderService fileReaderService;
+    private static final String NOT_EXIST_FILEPATH = "src/test/resources/invalid.csv";
+    private static final String EMPTY_FILEPATH = "src/test/resources/empty.csv";
+    private static final String VALID_FILEPATH = "src/test/resources/valid.csv";
```

Comment:
Check file path naming. What is file and what is filepath

Reply:
done